linera-base/src/crypto/ed25519.rs

```diff
@@ -36,6 +36,8 @@ pub struct Ed25519Signature(pub dalek::Signature);
 impl Ed25519SecretKey {
     #[cfg(all(with_getrandom, with_testing))]
     /// Generates a new key pair.
+    ///
+    /// Uses `OsRng` for that. If you want control over the RNG, use `generate_from`[Ed25519SecretKey::generate_from].
```

Comment:
nit: `/// Generates a new key pair using the operating system's RNG. [..]`

Reply:
```suggestion
    /// Generates a new key pair using the operating system's RNG.
    ///
    /// If you want control over the RNG, use `generate_from`[Ed25519SecretKey::generate_from].
```